view/readme.md

```diff
@@ -67,9 +68,9 @@ In the example we create a list with navigation (using $mol_nav)
 
 ## *.view.tree
 
-*view.tree* - is a declarative language of describing components, based on [format tree](https://github.com/nin-jin/tree.d). In a file could be plenty of components defined in series, but better way is put every component in a separate file, except very trivial cases.
-To create a new component it's enough to inherit this from any existing one.
-Names of the components should begin with `$` and be unique globally accordance with principles presented on [MAM](https://github.com/eigenmethod/mam). For example, let's declare the component `$my_button` extended from `$mol_view`:
+*view.tree* - is a declarative language of describing components, based on [tree format](https://github.com/nin-jin/tree.d). One file can have multiple component definitions, but better to put every component in a separate file, except in very trivial cases.
+To create a new component in `view.tree` file you must inherit it from any existing one or `$mol_view`.
+Name of the component should begin with `$` and be unique globally accordance with principles presented on [MAM](https://github.com/eigenmethod/mam). For example, let's declare the component `$my_button` extended from `$mol_view`:
```

Comment:
@nin-jin Please review changes in this file, some sentences are rephrased or simplified.